src/components/UpdatingForm/UpdatingForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  title: string,
+  id: number,
+  deleteTodo: (todoId: number) => void,
+  updateTodo: (
+    todoId: number,
+    newTodoData: Partial<Pick<Todo, 'title' | 'completed'>>
+  ) => void,
+  setIsUpdating: (isUpdating: boolean) => void;
+};
+
+export const UpdatingForm: React.FC<Props> = ({
```

Comment:
When I double click on the todo, input is not in focus and I need to click one more time on input to start typing

![image](https://github.com/mate-academy/react_todo-app-with-api/assets/84296041/1963d89e-2d65-40d6-95dd-3104f6f0c3a6)